File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/security/DeliusAuthenticationServiceException.java
package uk.gov.justice.digital.hmpps.oauth2server.security;

import org.springframework.security.authentication.InternalAuthenticationServiceException;

public class DeliusAuthenticationServiceException extends InternalAuthenticationServiceException {
    public DeliusAuthenticationServiceException() {
        super("Delius is currently not responding");
    }
}

File: src/main/java/uk/gov/justice/digital/hmpps/oauth2server/security/DeliusAuthenticationServiceException.kt
package uk.gov.justice.digital.hmpps.oauth2server.security

import org.springframework.security.authentication.InternalAuthenticationServiceException

class DeliusAuthenticationServiceException(username: String)
  : InternalAuthenticationServiceException(
    "Unable to retrieve information for $username from Delius.  We are unable to connect to Delius or there is an issue with $username in Delius")
